src/partials/integrations/disqus.html

```diff
@@ -15,25 +15,23 @@
 {% if not page.is_homepage and disqus %}
 {% set pageID = page.title | default("404", true) %}
 <h2 id="__comments" data-no-instant>{{ lang.t("meta.comments") }}</h2>
-<form id="gitalk-form" onsubmit="return false;" data-no-instant>
-<div id="gitalk-container" data-no-instant>
+<form id="comments-form" onsubmit="return false;" data-no-instant>
+<div class="giscus" id="comments-container" data-no-instant>
 </div>
 </form>
-<link rel="stylesheet" href="{{ 'assets/vendor/gitalk/gitalk.css' | url }}">
-<script src="{{ 'assets/vendor/gitalk/gitalk.min.js' | url }}"></script>
-<!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/OI-wiki/gitalk@master/dist/gitalk.css">
-<script src="https://cdn.jsdelivr.net/gh/OI-wiki/gitalk@master/dist/gitalk.min.js"></script> -->
-<script>
-  const gitalk = new Gitalk({
-    id: '{{ page.title | default("404", true)  }}',
-    owner: 'OI-wiki',
-    repo: 'gitment',
-    admin: ['Ir1d', 'cjsoft', 'TrisolarisHD', 'Xeonacid', 'sshwy', 'Enter-tainer', 'StudyingFather'],
-    clientID: 'd6a911c8fba0194626d4',
-    clientSecret: '867ec7e13cc99b420bd147cbb62d5cfec271ba81',
-    distractionFreeMode: false,
-    pagerDirection: 'first'
-  })
-  gitalk.render('gitalk-container')
+<script src="https://giscus.app/client.js"
+        data-repo="littlefrogfromthenorth/gisgus-comments"
+        data-repo-id="R_kgDOHwYTIQ"
+        data-category="Announcements"
+        data-category-id="DIC_kwDOHwYTIc4CQkz1"
+        data-mapping="url"
```

Comment:
```suggestion
        data-mapping="specific"
        data-term="{{ page.title | default("404", true) }}"
```

好像上面设了一个 `pageID` 变量，只不过原来 Gitalk 就没用 🤔

Reply:
pageID 没用, 只能从他给那几个里面选

Reply:
> pageID 没用

我是说可以用这个 HTML 模板的 `pageID` 变量代替 `{{ page.title | default("404", true) }}`，不知道你指的是什么..

> 只能从他给那几个里面选

这个就是选的 `specific` 然后再指定 `data-term` 啊

Reply:
![image](https://user-images.githubusercontent.com/49180735/182522430-c9141531-1226-4ec7-a251-25234ab6a21b.png)

Reply:
咱好像就不应该选specific

Reply:
特定 discussion 号是 number 不是 specific

Reply:
哦哦哦, 我看错了